protocols/mdns/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 0.33.1 [unreleased]
+
+- Use a random alphanumeric string for mDNS peer name (see [PR 2311]).
```

Comment:
```suggestion
- Use a random alphanumeric string instead of the local peer ID for mDNS peer
  name (see [PR 2311]).

  Note that previous versions of `libp2p-mdns` expect the peer name to be a
  valid peer ID. Thus they will be unable to discover nodes running this new
  version of `libp2p-mdns`.
```

Reply:
Do I need to correct the `protocols/mdns/CHANGELOG` to 0.34.0 as well here?

Reply:
Yes please :pray: